lib/components/input-group-addon.vue

```diff
@@ -0,0 +1,17 @@
+
+<template>
+    <span class="input-group-addon" :id="id">
+        <slot></slot>
+    </span>
+</template>
+
+<script>
+    export default {
+        props: {
+            id: {
+                type: String,
+                default: 'b_input_geoup_addon_' + this._uid
```

Comment:
Ech component has a unique ID on the vue VM instance as _uid

It will be needed if one wants to use aria-labeledby when referencing the addon(s).

But I can remove it and make the user enter an ID if needed.